modules/assim.sequential/inst/RemoteLauncher/Remote_sync.R

```diff
@@ -0,0 +1,72 @@
+#----------------------------------------------------------------
```

Comment:
Is this meant as an example?  Or a user would source this file from the package on-demand?